go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/operator-framework-olm
 
-go 1.22.5
+go 1.22.7
```

Comment:
@tmshort Why we are updating the golang version?

Reply:
It's required by the distribution update.
This is OK, because release-4.18 uses 1.22.9.

Reply:
See
https://github.com/distribution/distribution/blob/1b01625dae6a0a6321dadde7a30d0c3b92093a3e/go.mod#L3